crates/db_schema/src/source/tagline.rs

```diff
@@ -32,7 +32,16 @@ pub struct Tagline {
 #[derive(Clone, Default)]
 #[cfg_attr(feature = "full", derive(Insertable, AsChangeset))]
 #[cfg_attr(feature = "full", diesel(table_name = tagline))]
-pub struct TaglineForm {
+pub struct TaglineInsertForm {
+  pub local_site_id: LocalSiteId,
+  pub content: String,
+  pub updated: Option<DateTime<Utc>>,
+}
+
+#[derive(Clone, Default)]
+#[cfg_attr(feature = "full", derive(Insertable, AsChangeset))]
+#[cfg_attr(feature = "full", diesel(table_name = tagline))]
+pub struct TaglineUpdateForm {
   pub local_site_id: LocalSiteId,
   pub content: String,
   pub updated: Option<DateTime<Utc>>,
```

Comment:
Why would it need two options to make it nullable? Shouldn't one suffice?

Reply:
Is it possibly to be able to set it to `Some(None)`, effectively setting the `updated` column to `NULL`. Where as if you would have a `Option<DateTime<Utc>>` and set it to `None` it would be ignored (no updated) rather than set to `NULL`?